apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2Test.java

```diff
@@ -759,4 +791,46 @@ static void assertPartDMatches(Beneficiary inBeneficiary, Coverage inCoverage) {
     verifyRelationship();
     verifyPayor();
   }
+
+  static void checkForNoYearlyDate(Beneficiary inBeneficiary, Coverage inCoverage) {
+    // dual_01 thru dual_12
+    for (int i = 1; i < 13; i++) {
+      String url = String.format("https://bluebutton.cms.gov/resources/variables/dual_%02d", i);
+      verifyCodedExtensionDoestNotExist(inCoverage, url);
+    }
+
+    for (int i = 1; i < 13; i++) {
+      String url = String.format("https://bluebutton.cms.gov/resources/variables/buyin%02d", i);
+      verifyCodedExtensionDoestNotExist(inCoverage, url);
+    }
+
+    // ptdcntrct01 thru ptdcntrct12
+    for (int i = 1; i < 13; i++) {
+      String url = String.format("https://bluebutton.cms.gov/resources/variables/ptdcntrct%02d", i);
+      verifyCodedExtensionDoestNotExist(inCoverage, url);
+    }
+
+    // ptdpbpid01 thru ptdpbpid11
+    for (int i = 1; i < 12; i++) {
+      String url = String.format("https://bluebutton.cms.gov/resources/variables/ptdpbpid%02d", i);
+      verifyCodedExtensionDoestNotExist(inCoverage, url);
+    }
+
+    // sgmtid01 thru sgmtid11
+    for (int i = 1; i < 12; i++) {
+      String url = String.format("https://bluebutton.cms.gov/resources/variables/sgmtid%02d", i);
+      verifyCodedExtensionDoestNotExist(inCoverage, url);
+    }
+
+    for (int i = 1; i < 13; i++) {
```

Comment:
can we get a comment on this block like the other ones?

Reply:
yes